addons/project/controllers/portal.py

```diff
@@ -162,6 +162,9 @@ def _prepare_project_sharing_session_info(self, project, task=None):
         session_info.update(
             cache_hashes=cache_hashes,
             action_name="project.project_sharing_project_task_action",
+            action_context={
+                'allow_milestones': project.allow_milestones,
+            },
```

Comment:
@xavierbol Since the action is no longer a python method, but an xml record, I passed the context inside the session. What do you think ?